scripting/shavit-zones.sp

```diff
@@ -524,8 +535,15 @@ public void SQL_RefreshZones_Callback(Database db, DBResultSet results, const ch
 	while(results.FetchRow())
 	{
 		MapZones type = view_as<MapZones>(results.FetchInt(0));
-
-		if(type >= Zone_Freestyle)
+		
+		if(type == Zone_CustomSpawn)
+		{
+			gF_CustomSpawn[0] = results.FetchFloat(12);
+			gF_CustomSpawn[1] = results.FetchFloat(13);
+			gF_CustomSpawn[2] = results.FetchFloat(14);
+			gB_CustomSpawn = true;
+		}
+		else if(type >= Zone_Freestyle)
```

Comment:
Please make the spacing identical to the rest of the code (one new line before `else`)

Also, might want to use `bool EmptyZone(float vZone[3])` instead of using `gB_CustomSpawn`